### ISSUE TITLE
[accessibility]: add copy to the clipboard option to the codesnippets. 

### ISSUE THREAD

**Issue by nisarhassan12** [gitpod-io/website#2313]
![image](https://user-images.githubusercontent.com/46004116/175899260-bf06e34e-36a0-47d6-b87e-89622277c0df.png)

Thanks, @Siddhant-K-code for bringing this up [internally](https://gitpod.slack.com/archives/C01KGM9ETU6/p1656316948234189) on slack.

**Comment by Cahllagerfeld** [comment]
Hey, @nisarhassan12 I'd love to take that issue, as I looked into that before already. I'll assign it myself if thats okay for you 👍
